cmd/link/ocmrole/cmd.go

```diff
@@ -151,6 +151,7 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 		}
 		r.Reporter.Errorf("Unable to link role arn '%s' with the organization id : '%s' : %v",
 			roleArn, orgAccount, err)
+		cmd.SilenceUsage = true
```

Comment:
Is it worth setting this when defining the cmd :thinking: just wondering what is the use case for when we  do not want to silence the usage :thinking:

Reply:
It'll only be silenced for that error the help prompt should still show up when putting in invalid params